zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/AuthorizationCheckBehaviorTest.java

```diff
@@ -158,6 +158,49 @@ public void shouldGetResourceIdentifiersWhenRoleHasPermissions() {
     assertThat(resourceIdentifiers).containsExactlyInAnyOrder(resourceId1, resourceId2);
   }
 
+  @Test
```

Comment:
❌ Can we also add test cases in this class for the `getAuthorizedTenantIds` method?